mono/tests/Makefile.am

```diff
@@ -65,9 +95,58 @@ STRESS_TESTS_SRC=	\
 	process-stress.cs	\
 	assembly-load-stress.cs
 
+BAS_TEST_MOBILE_STATIC_BLACKLIST= \
```

Comment:
typo: ~~BAS_~~ BASE_

minor nit: maybe call it BASE_TEST_MOBILE_STATIC_NOT_SUPPORTED. Blacklist for me implies there's some sort of active filtering going on, but it's just a separate list that's used or not.